Formula/sqlline.rb

```diff
@@ -0,0 +1,18 @@
+class Sqlline < Formula
+  desc "Command-line shell for issuing SQL to relational databases via JDBC"
+  homepage "https://github.com/julianhyde/sqlline"
+  url "https://search.maven.org/remotecontent?filepath=sqlline/sqlline/1.11.0/sqlline-1.11.0-jar-with-dependencies.jar"
+  sha256 "725dbbe1e6b42399ad3741c522c8351d706c0b06623390ad425729f82901d4da"
+  license "BSD-3-Clause"
+
+  depends_on "openjdk"
+
+  def install
+    libexec.install "sqlline-#{version}-jar-with-dependencies.jar"
+    bin.write_jar_script libexec/"sqlline-#{version}-jar-with-dependencies.jar", "sqlline"
+  end
+
+  test do
+    assert_match "sqlline version #{version}\n", shell_output("#{bin}/sqlline -e '!quit' 2>&1")
```

Comment:
any chance that we can write a more functional test? or maybe sqline config test?

Reply:
@chenrui333 Will do. Any suggestions as far as what to test?

Reply:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.